frontend/src/pages/list/list.rs

```diff
@@ -89,10 +93,9 @@ pub fn update(
 }
 
 pub fn view(model: &Model, ctx: &Context) -> Node<Msg> {
-      
-
-    let filtered_lists = model.lists.iter()
-                                    .filter(|l| l.id.to_lowercase().contains(&model.filter))
+    let ctx_lists: Vec<crate::model::StoredList> = ctx.lists.values().cloned().collect();
```

Comment:
Not sure of the performance impacts of this. How is `model.lists` populated? I couldn't find it in the code...

Reply:
you can omit to collect... Just 
```rust
let filtered_lists = ctx.lists.values().
                                    .filter(|l| Uuid::to_string(&l.id).to_lowercase().contains(&model.filter))
                                    .map(|l| l.clone())
                                    .collect::<Vec<StoredList>>();
```